src/pages/hegel/reference/becoming/sublation.mdx

```diff
@@ -0,0 +1,205 @@
+which it derives_", which can be understood as a genealogy of its dialectical
+development. Put differently, in being sublated, the matter in question is no
+longer solely grasped in its own terms, but included in the terms of another.
+
+For example, to say that `being` is sublated in `becoming` means that `being` no
+longer is understood in its immediacy but as a moment of the context of
+`becoming`.
+
+But why is something sublated? Because a dialectic movement took place where
+something no longer is the something it immediately presented itself. Put
+differently, the matter at hand is not what it is. However, dialectical movement
+alone is not sufficient to understand something as sublated. `Being` vanishing
+into `nothing` does not thereby sublate `being`. To be something sublated, by
+contrast, entails a mutated state of the category that contains its immediate
+state as negated.
+
+#### Diaception (Niklas)
+
+In sublation, a double view necessarily exists. On the one hand, sublation
+expresses that a concept has turn out to be a _moment_ of another. On the other
+hand, however, sublation also expresses a trace to a past conception; putting it
+terse, a _concept_ has turned out to be a moment of another. One can understand
+this difference in terms of scopes: an absolute scope and a relative scope.
+
+Consider the example of `being`. In its sublation, `pure being` is rendered,
+paradoxically, a moment. A moment in the conceptual evolution of `being` whereby
+it was initially simple and pure but is now one among many.
+
+In more practical terms, that something is sublated means that it longer retains
+the absolute conceptual scope at hand but is relegated to a certain part of it.
+However, being in this partial state does not eliminate its initial immediacy
+altogether but keeps it precisely as negated. The immediacy _is_ negated, but
+the trace back to it is retained.
+
+The former thus holds a relative scope and the latter currently holds the
+absolute scope. Despite holding a relative scope, `being` nonetheless contains
+the trace to its original immediacy, and so we can say that it "points" back to
+its immediacy. And it can only "point" to this immediacy because it is exactly
+unavailable to it (were `being` not sublated, it would be immediate and there
+would be no mediation where any "pointing" would take place).
+
+To be something sublated, by contrast, entails a binding between its immediate
+state and its mediated state, or between its absolute- and relative scope.
+Diaception is the grasping of this binding, or holding in view both the absolute
+and relative scope of a category.
+
+In the context of `existence`, for example, where `being` and `nothing` are
+understood to be in a unity, `being` has its immediate state negated&mdash;there
+is no `pure being`&mdash;but in being _negated_ it is no less kept. Sublation
+entails diaception and thus implies development of a category.
```

Comment:
My comments about diaception aside, this is a very helpful section for explaining sublation to a novice.